starter-code/js/index.js

```diff
@@ -1,11 +1,60 @@
-var $cart = document.querySelector('#cart tbody');
-var $calc = document.getElementById('calc');
+var $cart = document.querySelector("#cart tbody");
+var $calc = document.getElementById("calc");
 
 function updateSubtot($product) {
+  let unitPrice = parseFloat($product.querySelector(".pu span").innerHTML);
+  let quantity = $product.querySelector(".qty label input").value;
+
+  const subTotal = unitPrice * quantity;
+
+  $product.querySelector(".subtot span").innerHTML = subTotal;
+  console.log(subTotal);
+  return subTotal;
+
   // Iteration 1.1
 }
 
-function calcAll() {
+function calcAll(price) {
   // Iteration 1.2
+  let totalPrice = 0;
+
+  document.querySelectorAll(".product").forEach(function(row) {
+    totalPrice += updateSubtot(row);
+  });
+
+  console.log(totalPrice);
+  let button = document.querySelector("h2>span");
```

Comment:
The variable name "button" is a bit misleading here as this is not a "button" but just plan text showing the total price. Making good variable names is super important when you start working in a company with large projects